BUILDER_DEV.md

```diff
@@ -1,390 +1,294 @@
+* Navigate to http://localhost:3000/#/pkgs
+* Log in
+* Click on "Create New Origin"
+* Fill out form (call the origin "core"), click "Save & Continue"
+* Click on "Keys"
+* Download the public and private key
+* Copy the public key contents
+* From a terminal, run
 
-Now, modify the `Procfile` (located in your hab repo in the `support` folder) to point the api, sessionsrv, jobsrv and worker services to the previously created config files, e.g.
+```
+$ hab origin key import
+```
+
+* Paste the public key contents
+* Hit Ctrl + D twice to close the stream
+* Run this again:
 
 ```
-api: target/debug/bldr-api start --config /home/your_alias/habitat/config_api.toml
-sessionsrv: target/debug/bldr-sessionsrv start --config /home/your_alias/habitat/config_sessionsrv.toml
-worker: target/debug/bldr-worker start --config /home/your_alias/habitat/config_worker.toml
-jobsrv: target/debug/bldr-jobsrv start --config /home/your_alias/habitat/config_jobsrv.toml
+$ hab origin key import
 ```
 
-## Build the Builder services for the first time
-1. Open a new terminal window.
-1. Run `make build-srv`
+* Paste the private key contents
+* Hit Ctrl + D twice to close the stream
 
-Note: this only needs to be done the first time to get the pieces in place. If you clean the project, make sure to unset the variables show below when you do a clean build, otherwise the build will look for packages on your local build service!
+### Install Dependencies in your local Builder Env
 
-## Run the Builder services
-1. Open a new terminal window.
-1. Export the following environment variables:
+* on your VM, go to the production instance of Builder (the one at habitat.sh)
+* Click on "My origins"
+* Click on "core"
+* Download public and private keys for the "core" origin (make sure the timestamp on the public key matches the one on the private key!) - we currently need the keys for the core production origin in order to upload some dependencies to our local Builder env)
+* From a terminal, run
 
 ```
-export HAB_AUTH_TOKEN=<your github token>
-export HAB_BLDR_URL=http://localhost:9636
-export HAB_ORIGIN=<your origin>
+$ hab origin key import
 ```
 
-Now run `sudo -E make bldr-run` from the root of your hab repo.
+* Paste the production core origin public key contents
+* Hit Ctrl + D twice
+* Run this again:
 
-The first time this command runs, it will create the required databases. Let it run for a while, and then re-start it if there are errors (this is normal for the first time setup).
+```
+$ hab origin key import
+```
 
-When `make bldr-run` can proceed with all the services up and running, you are ready to proceed to the next step.
+* Paste the production core origin private key contents
+* Hit Ctrl + D twice
+* Run
+```
+$ hab install core/hab-backline
+$ cd /hab/cache/artifacts
+$ hab pkg upload -c stable core-hab-backline...hart
```

Comment:
I think it'd be useful to mention here that if you get
```
✗✗✗
✗✗✗ No auth token specified
✗✗✗
```
That just means you need to add `HAB_AUTH_TOKEN="<your github token>"` to your environment.

Reply:
And if you get 
```
✗✗✗
✗✗✗ No such file or directory (os error 2)
✗✗✗
```
It means that the public key from `bldr.habitat.sh` (that corresponds to the private key the hart file was signed with) wasn't installed.